### PR TITLE
Making blob website responsive using grid

### PR DESCRIPTION
**Pull request by abhi1906singh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,118 @@
...
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="main">
+      <div class="wrapper">
+        <div class="navbar">
+          <div class="nav">
+            <div class="item1">Layout 3</div>
+            <div class="item2">About</div>
```
Comment (kishankumar0508) [inline index.html:22]: remove unnecessary class, if they are not used

File: index.html [hunk 2]
```diff
@@ -1,12 +1,118 @@
...
+            <div class="item3">Services</div>
```
Comment (kishankumar0508) [inline index.html:23]: remove unnecessary class, if they are not used

File: index.html [hunk 3]
```diff
@@ -1,12 +1,118 @@
...
+            <div class="item3">Contact</div>
```
Comment (kishankumar0508) [inline index.html:24]: remove unnecessary class, if they are not used

File: index.html [hunk 4]
```diff
@@ -1,12 +1,118 @@
...
+          </div>
+        </div>
+        <div class="sub_wrapper">
+            <div class="left">
+              <div class="blog_post_title">Blog Post Title</div>
+              <div class="by_test">by <inline class="test">Test</inline></div>
+
+            <div class="padding_date">
```
Comment (kishankumar0508) [inline index.html:32]: use related name of class name,
use genuine name of class name

File: index.html [hunk 5]
```diff
@@ -1,12 +1,118 @@
...
+                <p class="para4 para">
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Voluptates repudiandae consectetur adipisicing elit. Modi 
+                  veniam. Cupiditate  veniam itaque enim officiis
+                  nemo.
+                </p>
+              </div>
+            </div>
+            <!-- left starts here------------->
+            <div class="right">
```
Comment (kishankumar0508) [inline index.html:77]: use genuine name of class name

File: index.html [hunk 6]
```diff
@@ -1,12 +1,118 @@
...
+        <div class="navbar">
+          <div class="nav">
+            <div class="item1">Layout 3</div>
+            <div class="item2">About</div>
+            <div class="item3">Services</div>
+            <div class="item3">Contact</div>
+          </div>
+        </div>
+        <div class="sub_wrapper">
+            <div class="left">
```
Comment (kishankumar0508) [inline index.html:28]: use genuine name of class name

File: index.html [hunk 7]
```diff
@@ -1,12 +1,118 @@
...
+      <div class="wrapper">
+        <div class="navbar">
+          <div class="nav">
+            <div class="item1">Layout 3</div>
+            <div class="item2">About</div>
+            <div class="item3">Services</div>
+            <div class="item3">Contact</div>
+          </div>
+        </div>
+        <div class="sub_wrapper">
```
Comment (kishankumar0508) [inline index.html:27]: use genuine name of class name

File: index.html (hunk 2 above)
Comment (abhi1906singh) [reply index.html:23]: Okay sir

File: index.html [hunk 8]
```diff
@@ -1,12 +1,118 @@
...
+                      </div>
+            </div>
+             
+
+
+              <div class="img_div">
+                <div class="img">900 x 300</div>
+              </div>
+              <div class="paragraphs">
+                <div class="weight_para">
```
Comment (kishankumar0508) [inline index.html:45]: use genuine name of class name

File: index.html [hunk 9]
```diff
@@ -1,12 +1,118 @@
...
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi
+                  culpa doloremque impedit iste, praesentium dolore, consectetur
+                  sapiente dignissimos earum deserunt!
+                </div>
+                <p class="para1 para">
```
Comment (kishankumar0508) [inline index.html:50]: use genuine name of class name,
if para1 class is not used why unnecessary provided

File: index.html [hunk 10]
```diff
@@ -1,12 +1,118 @@
...
+                  Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quas
+                  fuga asperiores nisi ab consectetur adipisicing elit. Modi culpa
+                  doloremque ipsa modi minus eligendi
+                  voluptates.
+                </p>
+                <p class="para2 para">
```
Comment (kishankumar0508) [inline index.html:56]: if para2 class is not used why unnecessary provided

File: index.html [hunk 11]
```diff
@@ -1,12 +1,118 @@
...
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Voluptates repudiandae consectetur adipisicing elit. Modi culpa
+                  doloremque voluptatibus totam quo eos fugiat 
+                  nemo.
+                </p>
+                <p class="para3 para">
```
Comment (kishankumar0508) [inline index.html:62]: if para3 class is not used why unnecessary provided

File: index.html [hunk 12]
```diff
@@ -1,12 +1,118 @@
...
+                  Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quas
+                  fuga asperiores consectetur adipisicing elit. Modi culpa
+                  doloremque nisi ab ipsa modi minus eligendi
+                  voluptates.
+                </p>
+                <p class="para4 para">
```
Comment (kishankumar0508) [inline index.html:68]: if para4 class is not used why unnecessary provided

File: index.html [hunk 13]
```diff
@@ -1,12 +1,118 @@
...
+                <div class="blog_search">Blog Search</div>
+                <div class="input_div">
+                  <div class="input"><input type="text" /></div>
+                  <div class="search_bar">
+                    <i class="fa-solid fa-magnifying-glass"></i>
+                  </div>
+                </div>
+              </div>
+              <div class="blog_categories_div">
+                <div class="blog_categories">Blog Categories</div>
```
Comment (kishankumar0508) [inline index.html:88]: this class name can be blog_categories_title, it is more appropriate ,

File: index.html [hunk 14]
```diff
@@ -1,12 +1,118 @@
...
+                  <div class="blog_category2">
+                    <div class="item"><a href="#">Category name</a></div>
+                    <div class="item"><a href="#">Category name</a></div>
+                    <div class="item"><a href="#">Category name</a></div>
+                    <div class="item"><a href="#">Category name</a></div>
+                  </div>
+                </div>
+              </div>
+              <div class="side_widget_div">
+                <div class="side_widget">Side Widget Well</div>
```
Comment (kishankumar0508) [inline index.html:105]: this class name can be side_widget_title, it is more appropriate ,

**Pull request merged**